app/models/privacy_policy.rb

```diff
@@ -2,7 +2,7 @@
 
 class PrivacyPolicy < ActiveModelSerializers::Model
   DEFAULT_PRIVACY_POLICY = <<~TXT
```

Comment:
Unrelated to this PR -- thoughts on moving this big text/markdown block (or at least the portion of it that does not need string interpolation and is really just static text) out of this ruby file and into like `config/templates/privacy.md` or somewhere and have this constant be defined by reading the file? May be easier to review/edit as a markdown file instead of a ruby string in a class?

Reply:
Is it translatable in this location vs moving it to an external MD file?

Reply:
The code lower down checks for a value in site settings first, and then only defaults back to this if nothing is there. My guess would be that in the majority of installs where anyone actually cares about this language, that custom value gets set and read in.

Like I alluded to, it might be slightly more translatable/editable if moved out, because I bet the audience of "I'm comfortable editing a markdown file" people is larger than the "willing to edit a ruby file" people.

Reply:
I like the idea of having this in a separate file, feel free to open a PR doing it :)

Reply:
I have a branch that does this, will open as PR. There's one oddity to it with the string format/interpolation usage which I'll note on the PR.